docs/docs/config.md

```diff
@@ -0,0 +1,54 @@
+For multiple modules with common elements we can define a configuration to apply to all modules, or all modules of a certain type. Maintaining a config helps reduce the code repitition and allows you to change settings accross many widgets in one place. A config object can be passed as third, optional argument to `initializeWidgets`.
```

Comment:
Adding this pretty major feature to the very end of the docs almost feels like it's being reserved as a special treat for those who made it this far.

Maybe we need a generic API reference page or something that just spells out all the pathfora methods and their signatures.